src/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {Logo} from './logo.js'
+import {Toolbar} from './Toolbar.js'
+
```

Comment:
Update these statements, such lines only used when there are multiple components in same file.

Reply:
Done